.github/workflows/e2e.yml

```diff
@@ -73,6 +73,12 @@ jobs:
         kubectl logs -l name=resource-topology -c resource-topology-exporter-container || :
         kubectl get noderesourcetopologies.topology.node.k8s.io -A -o yaml
 
+    - name: deploy sample-devices
+      run: | 
+        hack/deploy-devices.sh
+        hack/check-ds.sh default device-plugin-a-ds
+        kubectl describe nodes -l node-role.kubernetes.io/worker= || :
```

Comment:
this means we create the devices before RTE starts, so when RTE starts it has the devices. I'm not sure this is the scenario we want to check - or, at least, that's not the ONLY scenario we want to check. We should also check the scenario on which devices are created after RTE started.

Reply:
That's exactly the scenario we have here.
Initially, we are running the RTE pods (in ```deploy RTE```), waiting for it to be ready (in ```cluster-ready```) and only then deploy the devices (in ```deploy sample-devices``)

Reply:
uhm, ok. This is good enough for a start